bbb/schemas/payload.yml

```diff
@@ -0,0 +1,77 @@
+id:          http://payload.bbb/v1.schema
```

Comment:
Ie. if you have somewhere you would like to publish it.. I recommend doing so.
Otherwise, let us grant you access to `schemas.taskcluster.net` (it's an S3 bucket) and you can put it there.